src/main/java/at/hannibal2/skyhanni/features/combat/BroodmotherFeatures.kt

```diff
@@ -0,0 +1,118 @@
+package at.hannibal2.skyhanni.features.combat
+
+import at.hannibal2.skyhanni.config.features.combat.broodmother.BroodmotherConfig.StageEntry
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.data.model.TabWidget
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.events.LorenzWorldChangeEvent
+import at.hannibal2.skyhanni.events.WidgetUpdateEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.HypixelCommands
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils.isInIsland
+import at.hannibal2.skyhanni.utils.RenderUtils.renderString
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import at.hannibal2.skyhanni.utils.SoundUtils
+import at.hannibal2.skyhanni.utils.SoundUtils.playSound
+import at.hannibal2.skyhanni.utils.TimeUtils.format
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.Duration.Companion.seconds
+
+@SkyHanniModule
+object BroodmotherFeatures {
+    val config get() = SkyHanniMod.feature.combat.broodmother
+
+    private var lastStage: String = ""
+    private var currentStage: String = ""
```

Comment:
can we maybe use the enum object directly for those two variables? to represent an empty string/no data known, dont invent a new enum stage but rather use null